### PR TITLE
RemoveTodo & rename Request.method's name

### PR DESCRIPTION
**Pull request by hyunsu15**
Remove Todo as i can 
1. rename Request.method's name
2. remove private static method because it is same public method.

don' t use first commit [refactor : AdRemoverAPI.HideViewWithLayout1dp in my opnion](https://github.com/revanced/revanced-integrations/pull/242/commits/d094d246dd9ae3590f2fb1924de7081d4e5e6afe)

it applies already.

### CODE REVIEW

File: app/src/main/java/app/revanced/integrations/sponsorblock/requests/SBRequester.java [hunk 1]
```diff
@@ -178,7 +188,8 @@ public static void retrieveUserStats(PreferenceCategory category, Preference loa
     public static void setUsername(String username, EditTextPreference preference, Runnable toastRunnable) {
         new Thread(() -> {
             try {
-                HttpURLConnection connection = getConnectionFromRoute(SBRoutes.CHANGE_USERNAME, SettingsEnum.SB_UUID.getString(), username);
+                HttpURLConnection connection = getConnectionFromRoute(SBRoutes.CHANGE_USERNAME,
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/sponsorblock/requests/SBRequester.java:191]: Theres lots of unrelated refactoring changes, please separate refactoring as a different commit

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: app/src/main/java/app/revanced/integrations/sponsorblock/requests/SBRequester.java (hunk 1 above)
Comment (hyunsu15) [reply app/src/main/java/app/revanced/integrations/sponsorblock/requests/SBRequester.java:191]: problem is my auto template save. i will correct it.
